streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamThreadTest.java

```diff
@@ -186,11 +187,23 @@ public boolean conditionMet() {
         assertEquals(thread.state(), StreamThread.State.DEAD);
     }
 
+    private Cluster createCluster(int numNodes) {
+        HashMap<Integer, Node> nodes = new HashMap<>();
+        for (int i = 0; i != numNodes; ++i) {
```

Comment:
`!=` feels a bit strange as opposed to `<` for a classic `for` loop's condition

Reply:
I just (lazily) reused it over from TopicAdminTest, but it is a bit non-standard. I'll change it there also.